js/viewer/annotation/annotation-group.js

```diff
@@ -57,6 +71,57 @@ function AnnotationGroup(id, anatomy, description, parent){
         return annotation;
     }
 
+    // this.addMarkerDef = function (stroke, subtype, checkExists = true) {
+    this.addMarkerDef = function (markerID, stroke, subtype, checkExists = true) {
+
+        // Check if the definition is already added
+        if (checkExists && document.querySelector("#" + markerID)) {
+            return;
+        }
+        var defs = document.createElementNS("http://www.w3.org/2000/svg", "defs");
+        var marker = document.createElementNS(
+        "http://www.w3.org/2000/svg",
+        "marker"
+        );
+        marker.setAttributeNS(null, "id", markerID);
+        marker.setAttributeNS(null, "markerUnits", "strokeWidth");
+        marker.setAttributeNS(null, "markerWidth", 10);
+        marker.setAttributeNS(null, "markerHeight", 10);
+        marker.setAttributeNS(null, "orient", "auto");
+        var arrowhead = null;
+        
+        switch(subtype){
+            case "solid": 
```

Comment:
If you found these definitions somewhere and used them, link to the original source. Otherwise explain what these are doing and why you're using these specific values.

Reply:
I came up with the name of the subtypes and the definitions however I found them from various sources. Some from blogs, some from official MDN docs. I've explained the attributes and their purpose in the comments.